.gitignore

```diff
@@ -1 +1,3 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.idea/
+package-lock.json
```

Comment:
this is not good unfortunately, lock should be versioned.

also .idea is something local to you, you should add that to a global ignore